wasm/js/animation_callback_handler.js

```diff
@@ -41,6 +76,18 @@ function AnimationCallbackHandler() {
         });
 
         this.onAfterCallbacks();
+
+        if (_fpsCounter) {
+            _fpsCounter.frameComplete();
+        }
+    }
+
+    this.enableFPSCounter = function(enable) {
```

Comment:
I wonder if we can make this a bit more flexible. what if we had a second parameter here called `options` which is an object. In the object, we could define `elementToDisplayTo` where the consumer can supply a DOM element to push the fps to. Then you don't have to deal with creating the element here and reasoning about styling.

Reply:
That sounds nice. We probably want to ultimately provide this to artists so they can see the fps of their animations, so it's worth making it look polished.

I also don't want to worry about making my own FPS element for a simple test page. Can we keep the simple fps counter  approach for when the object or elementToDisplayTo are not provided?